docs/jordan_gradient.rst

```diff
@@ -0,0 +1,16 @@
+Jordan Gradient
+==========================
+
+Overview
+--------
+
+This is an implementation of Stephen Jordan's "Fast Quantum Algorithm for
+Numerical Gradient Estimation" [1]_. It utilizes an oracle to sample some
+function at some distance, h, from the point of evaluation. These distances are
+encoded into the phase of an unitary operator. The ``phase_estimation`` algorithm
```

Comment:
You might want to say "in our implementation, the phase_estimation algorithm..." since he only mentions a QFT in his paper. (which is cheaper).